app/css/general.css

```diff
@@ -110,6 +114,14 @@ html,body {
 .button.current .icon{
 	-webkit-filter: invert(100%);
 }
+@keyframes fadeout {
+	from { opacity: 1; }
+	to   { opacity: 0; }
+}
+@keyframes fadein {
+	from { opacity: 0; }
+	to   { opacity: default; }
+}
```

Comment:
true. In this case I commented below to show the whole context. I guess it's a tradeoff.